contributing.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to AcadeMe
+
+Thank you for supporting AcadeMe and looking for ways to help. 
+We surely apperciate open source contributers! But before you contribute, please make sure to follow these guidelines.
+
+## Setting up
+1. Fork the repository! <img align="right" width="250" src="https://camo.githubusercontent.com/fcf9a4ed664cc63de2fcb14d1135072ba6d4c74a8e9bdb224ad6ab1e72600c3b/68747470733a2f2f6669727374636f6e747269627574696f6e732e6769746875622e696f2f6173736574732f526561646d652f666f726b2e706e67">
+2. Clone the repository!
+
+## Issues
+
+### Bug Reports
+
+We use GitHub [issues](https://github.com/redhat-beyond/AcadeMe/issues) to track public bugs. Report a bug by opening a new issue; it's that easy!
+Great Bug Reports tend to have:
+
+- A quick summary and/or background.
+- Steps to reproduce.
+- Be specific!
+- Give sample code if you can.
+- What actually happens.
+- Make it visual - use photos and screenshots if you can.
+
```

Comment:
another suggestion:
If the issue represents a feature you need to implement, write acceptance criteria for it.

Reply:
Can you give me a simple example? different features have different criteria

Reply:
meantime im pushing a new commit with the fixes of the suggestions above.

Reply:
> 

That's why the issue opener should write the acceptance criteria for his issue. 
The acceptance criteria are the description of what is expected from the solution in order for it to be accepted, in terms any contributor can understand and work with. 
simple example: `"The suggested function to solve it should get a string and an int as an argument and return a boolean"`

Reply:
also:
1. very nitpicky, but I think the `example` word in the `example` part that you added shouldn't be marked as code. 
2. still didn't link an issue to **THIS** pr

Reply:
Pushed the changes requested, also I noticed I cant link an issue to this PR through the Linked Issues section on this page. I tried doing it with keywords in the text like - Closes # 78
But I'm not sure if it worked.

Reply:
You have maintainer permissions so no, it is not related... What you need to do (as written in the contribution guide, and if not, should be added), is to add the closes/fix/resolve header to the original PR description (first message). try it

Reply:
I have just now seen your comment. I added the closes- #78 keywords to the first pr description
I have mistakenly thought that you requested adding it to the code